### PR TITLE
[Dubbo-3347] Update package name in README file  fix#3347

### PR DESCRIPTION
**Pull request by CrazyHZM**
## What is the purpose of the change

XXXXX

## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ There's a [README](https://github.com/dubbo/dubbo-samples/blob/master/dubbo-samp
 
 ```xml
 <properties>
-    <dubbo.version>2.6.5</dubbo.version>
+    <dubbo.version>2.7.1-SNAPSHOT</dubbo.version>
```
[the remark this answers is not in the document]
Comment (CrazyHZM) [reply README.md]: Is that still using 2.6.5?

[the remark this answers is not in the document]
Comment (ralf0131) [reply README.md]: Let's wait for 2.7.0 release. It will available next week.

[the remark this answers is not in the document]
Comment (CrazyHZM) [reply README.md]: Then I will temporarily change to 2.7.0 and wait for the release to be merged.

[the remark this answers is not in the document]
Comment (ralf0131) [reply README.md]: Looks good!

[the remark this answers is not in the document]
Comment (satansk) [reply README.md]: Can we use something like `latest.dubbo.version` to save us from updating it for every release?

[the remark this answers is not in the document]
Comment (ralf0131) [reply README.md]: I don't know if there is some fancy stuff here to help us. If you know please tell me :) Until we found something, we have to update it manually.

**Pull request merged**